### DISCUSSION TITLE
如果有「捏臉比賽」，人類只能拿第二名

### DISCUSSION THREAD

**Discussion by yenwei12138** [1101-GS4719-C/Python-Community-GS4719-C#523] [科技新知]
在許多科幻小說及電影中的未來世界，人類製造出的「複製人」和人類外形一模一樣，除非做情緒測試或找到隱藏的機型編號，根本無法辨認是否為「複製人」。雖然這對我們來說可能還有點遠，但如果只看臉的話，其實現在已經開始無法辨認真假人照了。

加州大學柏克萊分校 Hany Farid 教授近日發表於美國國家科學院院刊的研究顯示，現在 AI 合成人臉與真人沒有差別，甚至看起來比真人更值得信賴，且認為 AI 圖像合成技術發展和進步速度非常快，比傳統 CG 成像還快。

研究人員認為具一定可信度的 AI 合成照很有可能遭不法分子利用，在社群網路詐騙或引發混亂，需得到社會重視，且合成照技術發展需要約束。Farid 認為要解決這問題，唯一辦法就是為每張真人照加入「真實性」認證，其他人瀏覽和使用照片時才能辨認真實性。目前 Adobe、微軟等已在推廣相關技術。

https://technews.tw/2022/03/26/ai-synthesized-faces-are-indistinguishable-from-real-faces-and-more-trustworthy/

**Comment by dayu1004** [comment]
好好奇真實性認證是什麼，不知道傷疤算不算真實性認證的一種哈哈哈

**Comment by hapiwangy** [comment]
在現在造假的情形越來越多，的確是需要象徵真實的標籤。但要如何避免該標籤也被仿造又是另一個有趣的話題了

**Comment by JSHT** [comment]
透過區塊鏈來存原始圖檔確實是一項簡單可行的方案。不過不確定對於 AI 算法合成出的人臉與實際人臉，現今透過程式所觀察到的細微變化能不能精準分辨出兩者間的差異。

**Reply by JSHT** [reply]
目前像是微軟開發的工具能夠檢測影片或圖片是否經過後製、修圖。
而 adobe 則是為使用他們家產品處理過的圖片添加一個標籤標明這是一張處理過的圖片。

**Comment by ThomasMeng0515** [comment]
版主提供的連結好像有問題，沒辦法看~

**Comment by Jay-722** [comment]
這世上這麼多人不可能每個都真實性認證吧

**Comment by HuanYuuuuuu** [comment]
像是Deep fake這種東西就很容易造成假新聞。
用Deep fake來假造某個政治人物說話來帶動風向，而憑肉眼真的很難看出到底是不是假的

**Comment by xkllkx** [comment]
感覺在將來，真實性認證也會被破解

**Comment by Yuhsin000** [comment]
我覺得這個技術帶來的問題越來越多，希望日後隨著立法完善，可以減少這些問題

**Comment by henrywen547** [comment]
運用區塊鏈保留所有資訊似乎是個不錯的方法
但我也好奇是否這些資訊也能靠AI技術造假?

**Comment by sean1325418** [comment]
感覺以後會出現很多'不存在的人'做違法的事

**Comment by d26488399** [comment]
如果運用在影視作品的製作，應該有很好的效果

**Comment by Lynn12378** [comment]
太可怕了...真的完全看不出來是假人

**Comment by Sazmafuji** [comment]
雖然現在像deep fake的技術還沒有被禁用，但我認為此類技術可能之後都被禁用，雖然會用的人還是會繼續用

**Comment by leon10456** [comment]
感覺要讓法律跟上科技在使用 才不會有資安問題

**Comment by cordydeng** [comment]
不知道將來會不會導致真人無法被信任 , 反而被AI取代

**Comment by PoooYen** [comment]
就像刀子在不同的地方，可能是救人的工具，也可能是害人的東西...

**Comment by HuanYuuuuuu** [comment]
深偽技術用在資訊情報戰上感覺也會相當的可怕

**Comment by Lai27330** [comment]
ㄨㄚˊ居然完全沒有猜對，豪可怕，感覺這個人臉製造方法弊大於利，偽造假身分越來越簡單，會引發很多科技倫理的問題，是個使用前值得三思的新技術。
